### ISSUE TITLE
[Cheetah MVP] Add Plan Ahead page

### ISSUE THREAD

**Issue by laurenernest** [department-of-veterans-affairs/va.gov-team#19767]
## Issue Description
In the Cheetah POC, we had a "Project Cheetah booking information" page before starting the flow. Replace this with the new "Plan Ahead" page.

---
## Tasks
- [ ] Add Plan Ahead page between VAOS homepage and Screener page

## Acceptance Criteria
- [ ] Changes are behind va_online_scheduling_cheetah feature flag

- [ ] On click of "Learn more about COVID-19 vaccinations at the VA.", open Keep Me Informed page in new tab (/health-care/covid-19-vaccine/)

- [ ] On click of Start Scheduling CTA, go to Screener page

- [ ] ANALYTICS - Add tracking to Start Scheduling CTA

- [ ] CONTENT - Header and body content matches copy in specs

- [ ] DESIGN - Designs match specs https://www.sketch.com/s/e3493682-f69f-4910-a45c-36720e9639d9/a/ZOO8G73#Inspector

![Screen Shot 2021-02-10 at 10.46.46 AM.jpg](https://images.zenhubusercontent.com/5f7dd2dad6b9a9301bbab94e/90533706-ecb9-4397-ad30-12316c6b5b92)

### Out of Scope

- Common questions section
